hack/enable-kibana-msearch-access.sh

```diff
@@ -90,7 +90,11 @@ function curl_es_input() {
 
 usage() {
     echo Usage: $0 user-name [project-1 ... project-N]
+    echo  or
```

Comment:
Generally, LGTM, but have you thought about reusing https://github.com/openshift/origin-aggregated-logging/blob/master/elasticsearch/utils/logging for log message?